frontend/src/components/bucket/BucketsTable.vue

```diff
@@ -51,7 +51,7 @@ const showInfo = async (id: number) => {
         </div>
       </template>
     </Column>
-    <Column header="Actions" headerStyle="width: 12%" headerClass="header-right" bodyClass="content-right">
+    <Column header="Actions" headerStyle="width: 12%" headerClass="header-right" bodyClass="content-right actions-buttons">
```

Comment:
Any way to make this column always have a fixed size instead of relative? On relatively smaller viewports the buttons begin flowing around oddly.

Reply:
Fix added in PR-8